src/App/Document.cpp

```diff
@@ -4344,18 +4345,18 @@ Document::getPathsByOutList(const App::DocumentObject* from, const App::Document
     size_t index_to = indexMap[to];
     Path tmp;
     std::vector<Path> all_paths;
-    DocumentP::findAllPathsAt(all_nodes, index_from, all_paths, tmp);
+    DocumentP::findAllPathsAt(all_nodes, index_from, all_paths, std::move(tmp));

```

Comment:
Correct me if I'm wrong but can't we create the path object here instead and skip the move?
```suggestion
    DocumentP::findAllPathsAt(all_nodes, index_from, all_paths, Path());
```

Reply:
Yes, that will work. I'll change it.